PowerUp/app/src/main/java/powerup/systers/com/IntroActivity.java

```diff
@@ -0,0 +1,64 @@
+package powerup.systers.com;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+
+import com.github.paolorotolo.appintro.AppIntro;
+import com.github.paolorotolo.appintro.AppIntroFragment;
+
+public class IntroActivity extends AppIntro {
+
+    private SharedPreferences settings;
+    private boolean isfirstRun;
+
+    @Override
+    public void init(Bundle savedInstanceState) {
+
+        settings = getSharedPreferences("prefs", 0);
```

Comment:
It is confusing that the preferences are called `settings`. Could you give them a meaningful name?